geode-web-api/src/main/java/org/apache/geode/rest/internal/web/controllers/BaseControllerAdvice.java

```diff
@@ -65,8 +66,10 @@ protected String getRestApiVersion() {
   @ExceptionHandler({RegionNotFoundException.class, ResourceNotFoundException.class})
   @ResponseBody
   @ResponseStatus(HttpStatus.NOT_FOUND)
-  public String handle(final RuntimeException e) {
-    return convertErrorAsJson(e.getMessage());
+  public ResponseEntity<String> handle(final RuntimeException e) {
+    return ResponseEntity.status(HttpStatus.NOT_FOUND)
+        .contentType(APPLICATION_JSON_UTF8)
+        .body(convertErrorAsJson(e.getMessage()));
   }
```

Comment:
Could we we globally set a default **Content-Type**  without having to specify it in each API?

Reply:
Either that or a small helper method?!